src/Core/src/CommandMapperExtensions.cs

```diff
@@ -184,5 +184,81 @@ void newMethod(IElementHandler handler, IElement view, object? args)
 				action?.Invoke(handler!, view, args);
 			});
 		}
+
+		/// <summary>
+		/// Used internally to provide a fluent API for easily modifying
+		/// mappers at the point of instantiation.
+		/// Replace a command mapping in place but call the previous mapping if the types do not match.
+		/// </summary>
+		/// <typeparam name="TVirtualView">The cross-platform type.</typeparam>
+		/// <typeparam name="TViewHandler">The handler type.</typeparam>
+		/// <param name="commandMapper">The command mapper in which to change the mapping.</param>
+		/// <param name="key">The name of the command.</param>
+		/// <param name="method">The modified method to call when the command is updated.</param>
+		/// <returns>Target Command Mapper</returns>
+		internal static ICommandMapper<TVirtualView, TViewHandler> Replace<TVirtualView, TViewHandler>(this ICommandMapper<TVirtualView, TViewHandler> commandMapper,
```

Comment:
Similar to the property mappers, there are 2 overloads, one applies to the generic TView and the other applies to the IElement. The generic always replaces, the IElement does matching and fallback.